syne_tune/optimizer/baselines.py

```diff
@@ -77,7 +77,7 @@ def __init__(
 class PASHA(HyperbandScheduler):
     def __init__(
             self, config_space: Dict, metric: str, resource_attr: str,
-            **kwargs):
+            rung_system_kwargs: Dict, **kwargs):
```

Comment:
The idea here was to only expose really important args

Reply:
That makes sense - I've now changed it so that rung_system_kwargs is not exposed here - it can be accessed via kwargs, so it does not need to be there.